internal/services/automation/automation_software_update_configuration_resource.go

```diff
@@ -467,7 +467,7 @@ func (m SoftwareUpdateConfigurationResource) Arguments() map[string]*pluginsdk.S
 								"tag_filter": {
 									Type:     pluginsdk.TypeString,
 									Optional: true,
-									Computed: true,
+									Computed: !features.FourPointOhBeta(),
```

Comment:
Can we set an appropriate default here as well or does the API not return anything if this isn't set?

Reply:
the api doesn't appear to set anything for this but I added the FourPointOhBeta rather than just removing computed just erring on the side of caution